docs/src/content/docs/github-actions.md

```diff
@@ -0,0 +1,212 @@
+---
+title: GitHub Actions
+description: Automate releases and changelog previews with Craft GitHub Actions
+---
+
+Craft provides GitHub Actions for automating releases and previewing changelog entries in pull requests.
+
+## Prepare Release Action
+
+The main Craft action automates the `craft prepare` workflow in GitHub Actions. It creates a release branch, updates the changelog, and opens a publish request issue.
```

Comment:
You may want to refer to getsentry/publish repo as an example since we intend this repo to be useful to everyone, including people not working at Sentry.